_data/projects/OpenRCT2.yml

```
@@ -9,7 +9,7 @@ tags:
 - rollercoasters
 - 2d
 upforgrabs:
-  name: hacktoberfest
+  name: good first issue
   link: https://github.com/OpenRCT2/OpenRCT2/labels/hacktoberfest
```

This needs to be in sync with the label change

```suggestion
  link: https://github.com/OpenRCT2/OpenRCT2/labels/good%20first%20issue
```